lib/main.js

```diff
@@ -121,6 +121,12 @@ function initSession(shiny, options, shinyServer) {
 
       factory(url, ctx, (err, conn) => {
         pc.resolve(err, conn);
+        conn.onclose = e => {
+          if (!options.reconnect) {
+            log("connection closed; reconnect not enabled");
+            reconnectUI.showDisconnected();
```

Comment:
Hrm. Feels like we're missing doc to explain interactions / expectations between Shiny, this client, and the server.